shipment.go

```diff
@@ -135,6 +135,7 @@ type Shipment struct {
 	BatchID           string            `json:"batch_id,omitempty"`
 	BatchStatus       string            `json:"batch_status,omitempty"`
 	BatchMessage      string            `json:"batch_message,omitempty"`
+	TaxIdentifiers    []*TaxIdentifier  `json:"tax_identifiers,omitempty"`
```

Comment:
I think how we name the internal objects are somewhat inconsistent, and can be confusing.

It might make sense to introduce some kind of namespacing, like `EasyPost::TaxIdentifier` for the RHS value of the class/object.

https://github.com/EasyPost/easypost-go/issues/34